run_test.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+docker run -d --rm -p 7700:7700 getmeili/meilisearch:v0.13.0rc0 ./meilisearch  --no-analytics=true --master-key=5YsHgCa4hkA8W8hGrftkKPVbyQDHzdJR
```

Comment:
```suggestion
docker run -d --rm -p 7700:7700 getmeili/meilisearch:latest ./meilisearch  --no-analytics=true --master-key=masterKey
```

Reply:
We would rather use the latest version of MeiliSearch to avoid changing each time MeiliSearch does a release (we are currently to the v0.14.1)
And I would rather use `masterKey` as master key because we use this value in all the other SDKs, it avoids confusion for maintainers that work on different repositories 🙂

Reply:
Changed !